homeassistant/components/devolo_home_control/sensor.py

```diff
@@ -53,13 +53,19 @@ def __init__(self, homecontrol, device_instance, element_uid):
         self._device_class = DEVICE_CLASS_MAPPING.get(
             self._multi_level_sensor_property.sensor_type
         )
+
+        if self._device_class is None:
+            name = f"{device_instance.itemName} {self._multi_level_sensor_property.sensor_type}"
+        else:
+            name = device_instance.itemName
```

Comment:
Avoid the use of unnecessary else statements.

```suggestion
        name = device_instance.itemName
        if self._device_class is None:
            name += f" {self._multi_level_sensor_property.sensor_type}"           
```

Reply:
Thanks for the hint. Changed in latest commit.